olclient/openlibrary.py

```diff
@@ -1037,6 +1037,12 @@ def __init__(self, key, title=u"", subtitle=None, subject=None,
             work_olid = OpenLibrary._extract_olid_from_url(key, "works")
             edition_olids = edition_key
 
+            # clean author_names to compose result object:
+            # remove comma in an author's name if its not None only while retrieving data
+            if author_name:
```

Comment:
Deferring to @hornc on this one -- what do we do if the author's name accidentally contains multiple names which have been squished together by a comma? Can that happen? Also, how do we handle "first, last" versus "last, first"?

Reply:
There are tons of author records containing commas, some of them valid (, Jr., , MD, nobility titles, etc) and many of them not. While it's bad for an SDK/tool to throw an exception when reading data, it's worse for it to modify that data to be something different than what's in the database.

The checks should be on write, not read.